Solutions/Corvus.Monitoring.AspnetCore.Mvc.Specs/Fakes/TestOperationInstance.cs

```diff
@@ -0,0 +1,82 @@
+﻿// <copyright file="TestOperationInstance.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Corvus.Monitoring.AspnetCore.Mvc.Specs.Fakes
+{
+    using System.Collections.ObjectModel;
+    using Corvus.Monitoring.Instrumentation;
+
+    public class TestOperationInstance : IOperationInstance
+    {
+        public TestOperationInstance(string name, AdditionalInstrumentationDetail? additionalDetail = null)
+        {
+            this.Name = name;
+
+            if (additionalDetail?.MetricsIfPresent != null)
```

Comment:
I'm surprised the SDK didn't berate you for using `!= null` instead of `is not null` here.

This makes me wonder if we've not got diagnostics fully enabled on the test project.